js/src/modals/CreateAccount/createAccount.js

```diff
@@ -329,8 +329,6 @@ class CreateAccount extends Component {
   }
 
   onCreate = () => {
-    this.createStore.setBusy(true);
```

Comment:
Seems that we don't need to `setBusy(false)` as well (since it's done inside `createAccount` in store)